sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -512,6 +512,43 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils with Te
     }
   }
 
+  test("Save csv with custom charset") {
+    Seq("iso-8859-1", "utf-8", "windows-1250").foreach { encoding =>
+      withTempDir { dir =>
+        val csvDir = new File(dir, "csv").getCanonicalPath
+        // scalastyle:off
+        val originalDF = Seq("µß áâä ÁÂÄ").toDF("_c0")
+        // scalastyle:on
+        originalDF.write
+          .option("header", "false")
```

Comment:
The header flag is disabled by default. Just in case, are there any specific reasons fro testing without CSV header?

Reply:
My bad, there is no reason. It's fixed on the next commit.